src/supertux/menu/editor_delete_level_menu.cpp

```diff
@@ -53,7 +54,8 @@ EditorDeleteLevelMenu::menu_action(MenuItem& item)
       Dialog::show_message(_("You cannot delete level that you are editing!"));
     else
     {
-      Dialog::show_confirmation(_("Are you sure?"), [this, id]()
+      const std::string& level_name = get_item_by_id(id).get_text();
+      Dialog::show_confirmation(str(boost::format(_("You are about to delete level \"%s\". Are you sure?")) % level_name), [this, id]()
```

Comment:
Getting the button's label is poor practice; there is no guarantee that it won't change in the future. I'll try to find an alternative.